ambry-utils/src/test/java/com.github.ambry.utils/UtilsTest.java

```diff
@@ -100,9 +109,15 @@ public void testReadStrings()
     buffer.flip();
     outputString = Utils.readIntString(new DataInputStream(new ByteBufferInputStream(buffer)));
     Assert.assertEquals(s, outputString);
+    // 0-length
+    buffer.flip();
+    buffer.putInt(0);
+    buffer.rewind();
+    outputString = Utils.readShortString(new DataInputStream(new ByteBufferInputStream(buffer)));
+    Assert.assertEquals("", outputString);
```

Comment:
Minor: `assertTrue(outputString.isEmpty())` if you think it is clearer, otherwise disregard.